Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM node:12.10.0-alpine
+FROM node:14
 
 WORKDIR /app
 COPY . /app
 
-RUN npm install -g appcenter-cli@2.5.4 \
-    && apk update \
-    && apk add git \
-    && apk add bash
+RUN npm install -g appcenter-cli@2.5.4
+
+RUN mkdir -p /usr/local/lib/aapt/
+RUN wget -q "https://dl.google.com/dl/android/maven2/com/android/tools/build/aapt2/4.0.1-6197926/aapt2-4.0.1-6197926-linux.jar" -O aapt2-all.jar \
+    && mv aapt2-all.jar /usr/local/lib/aapt/aapt2-all.jar \
+    && cd /usr/local/lib/aapt \
+    && unzip -qq aapt2-all.jar \
+    && ln -s /usr/local/lib/aapt/aapt2 /usr/local/bin/aapt2
```

Comment:
According to the [documentation(https://developer.android.com/studio/command-line/aapt2#download_aapt2) to use aapt it is required to download the jar and extract it. The jar itself isn't executable 🤷

Reply:
Another thing here is the location where the `jar` file is extracted to. 
I could extract the file into the working directory as it gets bound as a volume (which makes me thinking if `COPY . /app` is actually needed if all the content gets immediately ovewritten 🤔)

Reply:
And one more thing is that I'm creating a symlink on a path that belongs to PATH as I wanted to make it executable everywhere, but for some reason I couldn't make it to work 🤷 If anyone has any idea why it didn't work in the bash script I'd love to know.